src/main/java/magicbees/main/utils/compat/BloodHelper.java

```diff
@@ -0,0 +1,87 @@
+package magicbees.main.utils.compat;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.IIcon;
+import net.minecraftforge.common.config.Configuration;
+import net.minecraftforge.common.config.Property;
+
+import WayofTime.alchemicalWizardry.ModItems;
+import WayofTime.alchemicalWizardry.api.altarRecipeRegistry.AltarRecipeRegistry;
+import cpw.mods.fml.common.Loader;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import magicbees.main.Config;
+import thaumcraft.api.ThaumcraftApi;
+import thaumcraft.api.aspects.Aspect;
+import thaumcraft.api.aspects.AspectList;
+import thaumcraft.common.config.ConfigItems;
+
+public class BloodHelper implements IModHelper {
+
+    @SideOnly(Side.CLIENT)
+    public static IIcon subtileIcons[];
+
+    private static boolean isBloodMagicActive = false;
+    public static final String Name = "AWWayofTime";
+
+    public static boolean isActive() {
+        return isBloodMagicActive;
+    }
+
+    public void preInit() {
+        if (Loader.isModLoaded(Name) && Config.bloodMagicActive) {
+            isBloodMagicActive = true;
+        }
+    }
+
+    public static void bloodMagicInstalled() {
+        isBloodMagicActive = true;
+    }
+
+    public void init() {
+        getItems();
```

Comment:
When you move the recipes in the getRecipes, gate this call with isBloodMagicActive check